FrameworkSystem/Client/MonitoringClient.py

```diff
@@ -85,7 +85,12 @@ def registerMonitoringClient(self, mc):
       self.__mcList.append(mc)
 
 
-gMonitoringFlusher = MonitoringFlusher()
+# DIRAC_USE_TORNADO_IOLOOP is defined by starting scripts
+if os.environ.get('DIRAC_USE_TORNADO_IOLOOP', 'false').lower() in ('yes', 'true'):
```

Comment:
This should be added to dirac-install.

Reply:
No it does not really need to be there, because it is only for services that are running with Tornado, so it is defined in the starting scripts